guides/common/modules/proc_extracting-gpg-public-key-fingerprints-from-release-files.adoc

```diff
@@ -36,9 +39,11 @@ $ gpg --verify Release.gpg Release
 +
 [options="nowrap" subs="+quotes"]
 ----
-$ gpg --armor --export 0146DC6D4A0B2914BDED34DB648ACFD622F3D138 A7236886F3CCCAAD148A27F80E98404D386FA1D9 A4285295FC7B1A81600062A9605C66F00D6C9793 > debian_11.txt
+$ gpg --armor --export A7236886F3CCCAAD148A27F80E98404D386FA1D9 4CB50190207B4758A3F73A796ED0E7B82643E131 A4285295FC7B1A81600062A9605C66F00D6C9793 > debian_11.txt
 ----
 +
+Ensure that `gpg` returns a `Good signature` message for each signature.
++
 Use this `.txt` file and upload it to {Project} as described below.
```

Comment:
Maybe rephrase this to:

Upload the `.txt` file to {Project}.
For more information, see xref:Importing_a_Custom_GPG_Key_{context}[import the custom GPG public key] into {Project}.